### DISCUSSION TITLE
How to choose profile for IMX6SX EVK of NXP

### DISCUSSION THREAD

**Discussion by Mdfayaz003** [OpenSCAP/openscap#2000] [Q&A]
I need to verify security policy for imx6sx board. I tried the following profile with the following command:
```
oscap xccdf eval --report basic-embedded.html /usr/share/oe-scap/OpenEmbedded_nodistro_0.xccdf.xml
```
And got the following output:
![image](https://github.com/OpenSCAP/openscap/assets/53575090/285d9969-b970-486d-a9d0-05878ee27f17)

But on trying the following command:
```
oscap xccdf eval --report basic-embedded.html --profile xccdf_org.ssgproject.content_profile_basic-embedded /usr/share/xml/scap/ssg/content/ssg-openembedded-ds.xml
```
All the cases results as **Not Applicable**
The result of the above command is attached below:
[result-log.zip](https://github.com/OpenSCAP/openscap/files/12027700/result-log.zip)

Am I missing any or else suggest the  proper profile and xml file to be use?

**Comment by sarnold** [comment]
I guess that depends on which version(s) you are using, as the patches for openembedded seem to be in flux right now.  In your first command above, I only see CVE output, but in your second command the profile doesn't match what I just saw in the latest OE patch (which is still in open [PR](https://github.com/ComplianceAsCode/content/pull/10793)).  I think your best bet for now might be trying the PR branch and running the scap-workbench on the xccdf file as shown in the docs.
